src/SerializerFactory.php

```diff
@@ -67,6 +69,13 @@ private function shouldUseObjectsForMaps() {
 	}
 
 	/**
+	 * @return bool
+	 */
+	private function shouldSerializeSnaksWithHash() {
+		return !(bool)( $this->options & self::OPTION_OBJECTS_FOR_MAPS );
```

Comment:
Oh... well, the cat is more interesting anyway. puh!

Anyway, the test passed, so all must therefore be good.

![nope-still-good](https://cloud.githubusercontent.com/assets/146040/8630921/c3af98ec-276c-11e5-9b1b-e15dd0ce79ba.jpg)